### PR TITLE
Implement About Pages

### PR DESCRIPTION
**Pull request by soalb-m**
## 📃 Description
Added an About page and a Licensees page 

## 🤔 Motivation and Context

It is required before publishing the app to Google Play Store

## 🧪 How Has This Been Tested?

Tested on real devices, and foldable emulators

## ⚒️ Does the PR contain new tests or refactored old ones?

Tests will come separate, in a different PR.

## 📷 Screenshots (if appropriate)

Screenshot_20221124_125745

## 📦 Types of changes

- [x] New feature (non-breaking change which adds functionality)
- [ ] Enhancement to a current functionality
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Bug fix (non-breaking change which fixes an issue. Please add issue link here too).

## ✅ Checklist

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] This PR contains new tests that cover the new code.
- [ ] This PR refactor previous tests to cover the new code.
- [ ] This PR is part of a set of PRs that build a bigger feature. If so, please, add here links to previously merged or open PRs.

### CODE REVIEW

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/BindingAdapters.kt [hunk 1]
```diff
@@ -0,0 +1,27 @@
+package com.microsoft.device.display.samples.duosamples.about
```
Comment (bimiron) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/BindingAdapters.kt:1]: This class needs copyright

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/ViewExtensions.kt [hunk 2]
```diff
@@ -0,0 +1,22 @@
+package com.microsoft.device.display.samples.duosamples.about
```
Comment (bimiron) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/ViewExtensions.kt:1]: This class needs copyright

**Review by bimiron — APPROVED** [review]
Looks good to me, just left two small comments

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamsAndLicensesFragment.kt [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+        }
+        binding?.licenseTermsTitle?.setOnClickListener {
+            onOssLicensesClicked()
+        }
+        setupDescriptionText()
+    }
+
+    private fun onOssLicensesClicked() {
+        activity?.let {
+            OssLicensesMenuActivity.setActivityTitle(getString(R.string.about_licenses_terms_title))
```
Comment (ancirja-m) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamsAndLicensesFragment.kt:70]: These two lines are duplicated on some files, you can extract an activity extansion function, like this:
Activity.openOssLicence() {
OssLicensesMenuActivity.setActivityTitle(getString(R.string.about_licenses_terms_title))
startActivity(Intent(this, OssLicensesMenuActivity::class.java))
}

and use it where you need like this: activity?.openOssLicence()

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamFragment.kt [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+        setupDescriptionText()
+    }
+
+    private fun setupDescriptionText() {
+        val clickableSpan = object : ClickableSpan() {
+            override fun onClick(view: View) {
+                startActivity(
```
Comment (ancirja-m) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamFragment.kt:47]: Also these lines can be extracted as an extension function, like this:

Activity.openUrl(url: String) {
                startActivity(
                    Intent(
                        Intent.ACTION_VIEW,
                       url
                    ).addFlags(Intent.FLAG_ACTIVITY_NEW_TASK)
                )
}

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/LayoutInfoViewModel.kt [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+ *
+ */
+
+package com.microsoft.device.display.samples.duosamples.about
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+
+class LayoutInfoViewModel : ViewModel() {
+    var isDualMode = MutableLiveData(false)
```
Comment (ancirja-m) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/LayoutInfoViewModel.kt:14]: To be honest, I didn't find the point from where this isDualMode var is updated.

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/AboutActivity.kt [hunk 6]
```diff
@@ -0,0 +1,96 @@
...
+                    }
+            }
+        }
+    }
+
+    /**
+     * Setup and adds fragments to the screen
+     */
+    private fun setupFragments(windowLayoutInfo: WindowLayoutInfo) {
+        if (windowLayoutInfo.isInDualMode()) {
```
Comment (ancirja-m) [inline SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/AboutActivity.kt:70]: What is happening when the app is in single screen? There is no behaviour defined in the code, is this correct?

File: SurfaceDuoSamples/src/main/res/values/styles.xml [hunk 7]
```diff
@@ -27,4 +29,50 @@
         <item name="colorAccent">@color/colorPrimary</item>
     </style>
 
+    <style name="Store" >
```
[the remark this answers is not in the document]
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/res/values/styles.xml:32]: Fixed

File: SurfaceDuoSamples/src/main/res/values-fr/strings_about.xml [hunk 8]
```diff
@@ -0,0 +1,43 @@
...
+    <string name="about_accessibility_blog">Microsoft Blog</string>
+    <string name="about_accessibility_twitch">Twitch</string>
+    <string name="about_accessibility_figma">Figma</string>
+    <string name="about_accessibility_learn">Microsoft Learn</string>
+    <string name="about_accessibility_twitter">Twitter</string>
+    <string name="about_accessibility_youtube">Youtube</string>
+
+    <string name="open_action_label">open</string>
+
+    <string name="docs_url">https://docs.microsoft.com/dual-screen/</string>
```
[the remark this answers is not in the document]
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/res/values-fr/strings_about.xml:33]: Fixed

File: SurfaceDuoSamples/src/main/res/values-de/strings_about.xml [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+    <string name="about_accessibility_blog">Microsoft Blog</string>
+    <string name="about_accessibility_twitch">Twitch</string>
+    <string name="about_accessibility_figma">Figma</string>
+    <string name="about_accessibility_learn">Microsoft Learn</string>
+    <string name="about_accessibility_twitter">Twitter</string>
+    <string name="about_accessibility_youtube">Youtube</string>
+
+    <string name="open_action_label">open</string>
+
+    <string name="docs_url">https://docs.microsoft.com/dual-screen/</string>
```
[the remark this answers is not in the document]
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/res/values-de/strings_about.xml:33]: Fixed

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/samples/SampleBuilder.kt [hunk 10]
```diff
@@ -9,19 +9,8 @@
 package com.microsoft.device.display.samples.duosamples.samples
```
[the remark this answers is not in the document]
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/samples/SampleBuilder.kt:9]: com.microsoft.device.display.samples.duosamples is the package name of the Bundle app. Not sure if we need to change it.
The last "samples" is a package where where I keep the list of samples from the Bundle app.

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamsAndLicensesFragment.kt (hunk 3 above)
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamsAndLicensesFragment.kt:70]: Fixed

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamFragment.kt (hunk 4 above)
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/fragments/AboutTeamFragment.kt:47]: Fixed

File: SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/AboutActivity.kt (hunk 6 above)
Comment (soalb-m) [reply SurfaceDuoSamples/src/main/java/com/microsoft/device/display/samples/duosamples/about/AboutActivity.kt:70]: It is defined in the layout xml  file

**Pull request merged**